core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -167,7 +168,7 @@ private[spark] class DAGScheduler(
   // Stages that must be resubmitted due to fetch failures
   private[scheduler] val failedStages = new HashSet[Stage]
 
-  private[scheduler] val activeJobs = new HashSet[ActiveJob]
+  private[spark] val activeJobs = new HashSet[ActiveJob]
```

Comment:
Use a listener to keep track of active jobs, and remove this please.

Reply:
Let us keep PR consistent, the change is small enough that we dont need to split it.

Reply:
Done! I reverted the change and switched to use `SparkListenerJobStart` to get the tags.